docs/stylesheets/extra.css

```diff
@@ -1,133 +1,23 @@
-  --md-code-bg-color:                hsla(var(--md-hue), 15%, 15%, 1);
-
-  /* Code highlighting color shades */
-  --md-code-hl-color:                hsla(#{hex2hsl($clr-blue-a200)}, 0.15);
-  --md-code-hl-number-color:         hsla(6, 74%, 63%, 1);
-  --md-code-hl-special-color:        hsla(340, 83%, 66%, 1);
-  --md-code-hl-function-color:       hsla(291, 57%, 65%, 1);
-  --md-code-hl-constant-color:       hsla(250, 62%, 70%, 1);
-  --md-code-hl-keyword-color:        hsla(219, 66%, 64%, 1);
-  --md-code-hl-string-color:         hsla(150, 58%, 44%, 1);
-  --md-code-hl-name-color:           var(--md-code-fg-color);
-  --md-code-hl-operator-color:       var(--md-default-fg-color--light);
-  --md-code-hl-punctuation-color:    var(--md-default-fg-color--light);
-  --md-code-hl-comment-color:        var(--md-default-fg-color--light);
-  --md-code-hl-generic-color:        var(--md-default-fg-color--light);
-  --md-code-hl-variable-color:       var(--md-default-fg-color--light);
-
-  /* Typeset color shades */
-  --md-typeset-color:                var(--md-default-fg-color);
-
-  /* Typeset `a` color shades */
   --md-typeset-a-color:              var(--eb-blue);
-
-  /* Typeset `a:hover` color shades */
   --md-accent-fg-color:              var(--eb-green);
-
-  /* Typeset `mark` color shades */
-  --md-typeset-mark-color:           hsla(#{hex2hsl($clr-blue-a200)}, 0.3);
-
-  /* Typeset `kbd` color shades */
-  --md-typeset-kbd-color:            hsla(var(--md-hue), 15%, 94%, 0.12);
-  --md-typeset-kbd-accent-color:     hsla(var(--md-hue), 15%, 94%, 0.2);
-  --md-typeset-kbd-border-color:     hsla(var(--md-hue), 15%, 14%, 1);
-
-  /* Typeset `table` color shades */
-  --md-typeset-table-color:          hsla(var(--md-hue), 75%, 95%, 0.12);
-
-  /* Admonition color shades */
-  --md-admonition-fg-color:          var(--md-default-fg-color);
-  --md-admonition-bg-color:          var(--md-default-bg-color);
-
-  /* Footer color shades */
-  --md-footer-bg-color:              hsla(var(--md-hue), 15%, 12%, 0.87);
-  --md-footer-bg-color--dark:        hsla(var(--md-hue), 15%, 10%, 1);
-
-  /* Shadow depth 1 */
-  --md-shadow-z1:
-    0 #{px2rem(4px)} #{px2rem(10px)} hsla(0, 0%, 0%, 0.2),
-    0 0              #{px2rem(1px)}  hsla(0, 0%, 0%, 0.1);
-
-  /* Shadow depth 2 */
-  --md-shadow-z2:
-    0 #{px2rem(4px)} #{px2rem(10px)} hsla(0, 0%, 0%, 0.3),
-    0 0              #{px2rem(1px)}  hsla(0, 0%, 0%, 0.25);
-
-  /* Shadow depth 3 */
-  --md-shadow-z3:
-    0 #{px2rem(4px)} #{px2rem(10px)} hsla(0, 0%, 0%, 0.4),
-    0 0              #{px2rem(1px)}  hsla(0, 0%, 0%, 0.35);
-
-  /* Hide images for light mode */
-  img[src$="#only-light"],
-  img[src$="#gh-light-mode-only"] {
-    display: none;
-  }
-
-  /* Show images for dark mode */
-  img[src$="#only-dark"],
-  img[src$="#gh-dark-mode-only"] {
-    display: initial;
-  }
 }
+[data-md-color-scheme="default"] {
+  /* EB colours - main logo border colours */
+  --eb-blue:  #72a4b0;
+  --eb-green: #8ca379;
```

Comment:
Both the blue and green is a bit too light on a white background imho, so let's use slightly darker variants, see https://github.com/branfosj/easybuild-docs/pull/2